tfjs-converter/python/tensorflowjs/converters/normalize_biasAdd.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Google LLC
```

Comment:
please use snake case for the filename
normalize_bias_add.py

Reply:
Done.